### PR TITLE
Add support for $skiptoken in the List group members

### PR DESCRIPTION
**Pull request by matthewbolanos**
---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [ ] The PR has modified HISTORY.rst describing any customer-facing, functional changes. Note that this does not include changes only to help content. (see [Modifying change log](https://github.com/Azure/azure-cli/tree/master/doc/authoring_command_modules#modify-change-log)).

### CODE REVIEW

File: certified-connectors/AzureAD/apiDefinition.swagger.json [hunk 1]
```diff
@@ -316,14 +316,13 @@
             "format": "int32"
           },
           {
-            "name": "$skip",
+            "name": "$skiptoken",
```
Comment (ShyamSunder90) [inline certified-connectors/AzureAD/apiDefinition.swagger.json]: How will user get this?

**Comment by schabungbam** [comment]
Skiptoken is not something that will come from end users. The entire nextLink is supposed to be treated as is - we are not supposed to parse that and extract the $skiptoken.
So, ideally, we will be handling the ODATA nextLink transparently, and enable pagination.

File: certified-connectors/AzureAD/apiDefinition.swagger.json (hunk 1 above)
Comment (schabungbam) [reply certified-connectors/AzureAD/apiDefinition.swagger.json]: Will not. See the other comment in the main PR thread. I also submitted a PR to revert the $skip token.

**Comment by dougelliott** [comment]
@schabungbam @ShyamSunder90 Can this PR be closed out in favor of #209?

**Comment by mamurshe** [comment]
This was already included in a later checkin.

**Pull request closed without merging**